src/clientManager.js

```diff
@@ -140,9 +140,10 @@ ClientManager.prototype.removeUser = function(name) {
 	return true;
 };
 
-ClientManager.prototype.autoload = function(/* sockets */) {
+ClientManager.prototype.autoload = function ClientAutoLoad() {
```

Comment:
I wanted to include the class name in the name. But in this case I messed up a little.